### PR TITLE
Add zh-Hant and zh-Hans translation for CSRF error page

### PR DESCRIPTION
**Pull request by andychow326**
ref DEV-1890

### CODE REVIEW

**Comment by louischan-oursky** [comment]
Is this ready for review?

File: pkg/lib/web/static_assets.go [hunk 1]
```diff
@@ -25,10 +25,10 @@ var Favicon = resource.RegisterResource(LocaleAwareImageDescriptor{Name: "favico
 var AppBackgroundImage = resource.RegisterResource(NonLocaleAwareImageDescriptor{Name: "app_background_image", SizeLimit: 500 * 1024})
 var AppBackgroundImageDark = resource.RegisterResource(NonLocaleAwareImageDescriptor{Name: "app_background_image_dark", SizeLimit: 500 * 1024})
 
-var CSRFIOSInsturction = resource.RegisterResource(StaticImageDescriptor{Name: "csrf-ios-instruction"})
-var CSRFAndroidInsturction = resource.RegisterResource(StaticImageDescriptor{Name: "csrf-android-instruction"})
-var CSRFChromeInsturction = resource.RegisterResource(StaticImageDescriptor{Name: "csrf-chrome-instruction"})
-var CSRFSamsungInsturction = resource.RegisterResource(StaticImageDescriptor{Name: "csrf-samsung-instruction"})
+var CSRFIOSInsturction = resource.RegisterResource(LocaleAwareImageDescriptor{Name: "csrf-ios-instruction"})
```
Comment (louischan-oursky) [inline pkg/lib/web/static_assets.go]: LocaleAwareImageDescriptor is for images that allow updates, like app logo.

Please
1. Rename StaticImageDescriptor to NonLocalAwareStaticImageDescriptor
2. Introduce LocalAwareStaticImageDescriptor which is static and locale aware.

File: pkg/lib/web/user_agent.go [hunk 2]
```diff
@@ -1,33 +1,53 @@
...
 type RecognizedMobileDeviceType string
 
+type RecognizedMobileDevice struct {
+	Type                 RecognizedMobileDeviceType `json:"type"`
+	OSVersionMajorString string                     `json:"os_version_major_string"`
+	OSVersionMinorString string                     `json:"os_version_minor_string"`
+	OSVersionPatchString string                     `json:"os_version_patch_string"`
+	OSVersionMajorInt    int                        `json:"os_version_major_int"`
+	OSVersionMinorInt    int                        `json:"os_version_minor_int"`
+	OSVersionPatchInt    int                        `json:"os_version_patch_int"`
```
Comment (louischan-oursky) [inline pkg/lib/web/user_agent.go:19]: Remove the JSON tags. We do not marshal this struct as JSON.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: pkg/lib/web/static_assets.go (hunk 1 above)
Comment (andychow326) [reply pkg/lib/web/static_assets.go]: ok

File: pkg/lib/web/user_agent.go (hunk 2 above)
Comment (andychow326) [reply pkg/lib/web/user_agent.go:19]: ok

**Comment by andychow326** [comment]
Hi @louischan-oursky, it is ready for review now. :adore2x:

Please help run `make generate-translations` as credential is required :adore2x:

**Comment by louischan-oursky** [comment]
I am closing this.

**Pull request closed without merging**